File: C++/selectionsort.cpp
#include <vector>
using namespace std;
vector<int> selectionSort(vector<int> array) {
  // Write your code here.
	int currindex= 0;
	while(currindex < array.size() -1){
		int minindex = currindex;
		for(int k=currindex +1;k<array.size();++k){
			if(array[minindex]>array[k]){
				int temp = array[minindex];
				array[minindex] = array[k];
				array[k] = temp;
				
			}
		}
		currindex++;
	}
	
	
	
  return {array};
}

File: Python/selectionsort.py
def selectionSort(array):
    # Write your code here.
	currindex = 0
	while currindex < len(array)-1:
		minindex = currindex
		for i in range(currindex + 1, len(array)):
			if array[minindex]>array[i]:
				minindex = i
		swap(currindex, minindex, array)	
		currindex +=1
    
    return array

def swap(i,j,array):
	array[i], array[j]=array[j],array[i]